Alexa.NET/Response/Directive/VoicePlayerSpeakDirective.cs

```diff
@@ -5,11 +5,14 @@
 
 namespace Alexa.NET.Response.Directive
 {
-    public class VoicePlayerSpeakDirective:IProgressiveResponseDirective
+    public class VoicePlayerSpeakDirective : IProgressiveResponseDirective
     {
-        public VoicePlayerSpeakDirective(Ssml.Speech speech):this(speech.ToXml())
+        public VoicePlayerSpeakDirective()
```

Comment:
The other parameterless constructors you've added have been internal - is there a reason this one is public?

Reply:
Nope just an oops on my part
```suggestion
        internal VoicePlayerSpeakDirective()
```